cmd/helm/inspect.go

```diff
@@ -184,16 +216,36 @@ func (i *inspectCmd) run() error {
 		return err
 	}
 
-	if i.output == chartOnly || i.output == both {
+	if i.output == chartOnly || i.output == all {
 		fmt.Fprintln(i.out, string(cf))
 	}
 
-	if (i.output == valuesOnly || i.output == both) && chrt.Values != nil {
-		if i.output == both {
+	if (i.output == valuesOnly || i.output == all) && chrt.Values != nil {
+		if i.output == all {
 			fmt.Fprintln(i.out, "---")
 		}
 		fmt.Fprintln(i.out, chrt.Values.Raw)
 	}
 
+	if i.output == readmeOnly || i.output == all {
```

Comment:
For charts without a readme file, if you don't specify `readme` subcommand you will still see 
```
---
No readme found in template
```
Please modify this to skip that output unless the `readme` subcommand is invoked. The other subcommands do not have a `not found` message so we could consider dropping it altogether.

Reply:
I will drop it out altogether